selene/src/main.rs

```diff
@@ -391,6 +391,55 @@ fn read_file(checker: &Checker<toml::value::Value>, filename: &Path) {
     );
 }
 
+/// Reads a config file and return content and path. None if not found or empty.
+fn read_config_file() -> Option<(String, Option<PathBuf>)> {
+    let config_paths_to_check = [".selene.toml", "selene.toml"];

```

Comment:
Should be a const.

Reply:
Closed in https://github.com/Kampfkarren/selene/pull/591/commits/822bc446494e5e856a9ae5df5ed0fd56fd9e0ebe